types/user.ts

```diff
@@ -115,15 +118,15 @@ export class User extends LazyObject {
    * @param content: the post or comment to be reported
    */
   public async addReport(content: Post | Comment | User) {
-
     // check that the content to be reported isn't from an employee/manager, we don't want to automod them
-    if (await (await content.getAuthor()).getStatus() >= UserStatus.Employee)
+    if ((await (await content.getAuthor()).getStatus()) >= UserStatus.Employee)
       return Promise.reject('Cant report an employees content');
 
     if (!content.hasData) await content.getData();
 
     // if already reported or already has a significant number of reports (arbitrary), return
-    if (await this.alreadyReported(content) || content.reports?.length! > 7) return;
+    if ((await this.alreadyReported(content)) || content.reports?.length! > 7)
```

Comment:
oh god. thank you for catching this

Reply:
I didn't, lol, the autoformatter did. You should turn that on lol